qemu/qemu.go

```diff
@@ -153,6 +162,9 @@ const (
 func (transport VirtioTransport) defaultTransport(config *Config) VirtioTransport {
 	switch runtime.GOARCH {
 	case "amd64", "386":
+		if config != nil && config.Machine.Type == MachineTypeMicrovm {
```

Comment:
Does this really also apply to 32-bit?

Reply:
microvm is supported by 32-bit QEMU - is that what would decide it?

```
$ i386-softmmu/qemu-system-i386 -machine help | grep microvm
microvm              microvm (i386)
```